src/staticData.js

```diff
@@ -109,4 +76,45 @@ const disabilityList = [
   }
 ];
 
+const reasonsList = [
+  {
+    key: "x",
+    text: "Wybierz rodzaj wniosku...",
+    value: "WYBIERZ_WNIOSEK",
+    disabilityListItems: disabilityList
+  },
+  {
+    key: "0",
+    text: "Kształcenie specjalne",
+    value: "SPECJALNE",
+    disabilityListItems: disabilityList.filter(
```

Comment:
Don't you think we should separate row data from business logic? Maybe It'd be better to move it to `Applicant` logic?